boards/xtensa/esp32/ttgo_t_display_esp32/configs/lvgl_fb/defconfig

```diff
@@ -43,7 +43,13 @@ CONFIG_LCD_ST7789_XOFFSET=53
 CONFIG_LCD_ST7789_XRES=135
 CONFIG_LCD_ST7789_YOFFSET=40
 CONFIG_LCD_ST7789_YRES=240
-CONFIG_LV_HOR_RES=135
+CONFIG_LV_DEMO_WIDGETS_SLIDESHOW=y
+CONFIG_LV_MEM_CUSTOM=y
```

Comment:
Please, add (from `menuconfig`) the following flags:
1. `CONFIG_LV_FONT_MONTSERRAT_12`
2. `CONFIG_LV_FONT_MONTSERRAT_18`